documentation/asciidoc/computers/configuration/boot_folder.adoc

```diff
@@ -75,7 +75,7 @@ The boot folder will contain various xref:linux_kernel.adoc#kernel[kernel] image
 | Beta xref:config_txt.adoc#boot-options[64 bit kernel]. Raspberry Pi 2 with BCM2836 does not support 64-bit.
 |===
 
-NOTE: The architecture reported by `lscpu` is `armv7l` for 32-bit systems (i.e. everything except kernel8.img), and `aarch64` for 64-bit systems. The `l` in the `armv7l` case refers to the architecture being little-endian, not `LPAE` as is indicated by the `l` in the `kernel7l.img` filename.
+NOTE: The architecture reported by `lscpu` is `armv7l` for systems running a 32-bit kernel (i.e. everything except kernel8.img), and `aarch64` for systems running a 64-bit kernel. The `l` in the `armv7l` case refers to the architecture being little-endian, not `LPAE` as is indicated by the `l` in the `kernel7l.img` filename.
```

Comment:
While you're at it, could you add backticks around `kernel8.img` please? :slightly_smiling_face:

Reply:
No problem - done.